src/Microsoft.Data.SqlClient/tests/Common/LocalAppContextSwitchesHelper.cs

```diff
@@ -28,6 +28,7 @@ public sealed class LocalAppContextSwitchesHelper : IDisposable
     private readonly PropertyInfo _useMinimumLoginTimeoutProperty;
     private readonly PropertyInfo _legacyVarTimeZeroScaleBehaviourProperty;
     private readonly PropertyInfo _useConnectionPoolV2Property;
+    private readonly PropertyInfo _ignoreServerProvidedFailoverPartner;
```

Comment:
#3625 had a bool `IgnoreServerProvidedFailoverPartner` and has a different getter. Is that intentional?